core/src/test/java/org/elasticsearch/bwcompat/BasicBackwardsCompatibilityIT.java

```diff
@@ -371,16 +380,13 @@ public void assertVersionCreated(Version version, String... indices) {
         }
     }
 
-
-    @Test
     public void testUnsupportedFeatures() throws IOException {
         XContentBuilder mapping = XContentBuilder.builder(JsonXContent.jsonXContent)
                 .startObject()
                     .startObject("type")
                         .startObject(FieldNamesFieldMapper.NAME)
-                           // by setting randomly index to no we also test the pre-1.3 behavior
-                            .field("index", randomFrom("no", "not_analyzed"))
-                            .field("store", randomFrom("no", "yes"))
+                            // by setting randomly enabled to no we also test the pre-1.3 behavior
+                            .field("enabled", randomFrom("no", "yes"))
```

Comment:
This is that internal field names field used for the _missing query. It isn't allowed to have `index` or `store` any more. But `enabled` replaces `index` in this case. `store` was weird, iirc. We didn't need it stored to use it so this was just twiddling that bit for the sake of it I think. Because it was an option.